.cirrus.yml

```diff
@@ -25,15 +25,14 @@ env:
 
     ####
     #### Cache-image names to test with (double-quotes around names are critical)
-    #### Comment out fedora-35 for podman 4.x branches.
     ####
-    FEDORA_NAME: "fedora-37"
-    FEDORA_AARCH64_NAME: "${FEDORA_NAME}-aarch64"
-    PRIOR_FEDORA_NAME: "fedora-36"
-    UBUNTU_NAME: "ubuntu-2204"
+    FEDORA_NAME: "fedora-37" ### c20230120t152650z-f37f36u2204
+    FEDORA_AARCH64_NAME: "${FEDORA_NAME}-aarch64" ### c20230120t152650z-f37f36u2204
+    PRIOR_FEDORA_NAME: "fedora-36" ### c20230120t152650z-f37f36u2204
+    #UBUNTU_NAME: "ubuntu-2204"  ### c20230120t152650z-f37f36u2204
```

Comment:
Are these nasty comments absolutely necessary? I've been reviewing this indirectly via #17305 and the constant changes in comment are distracting.

Reply:
Not absolutely necessary.  They're there because there's no other way (easy) way to associate a IMAGE_SUFFIX value with any of the *_NAME values.  In other words, I want maintainers to notice quickly if renovate suggests an incompatible update - i.e. replacing F37 with F38.

However, [the current rules](https://github.com/containers/automation/blob/main/renovate/defaults.json5#L136-L139) treat everything after the `-` as a "compatibility" value, so (assuming that works) the comment is just a "CYA" value that gets updated with images.  Hopefully prompting a maintainer to check - for example, yep, these images still cover "Fedora-36".

Maybe this is just a corner-case we can ignore though?

Reply:
I'll endeavor to remove these comments when I come across them.  Such as in #17305

Reply:
SGTM